src/sentry/notifications/utils/__init__.py

```diff
@@ -360,6 +360,11 @@ def get_parent_and_repeating_spans(
     return (parent_span, repeating_spans)
 
 
+def occurrence_perf_to_email_html(context: Any) -> Any:
+    """Generate the email HTML for an occurrence-backed performance issue alert"""
+    return render_to_string("sentry/emails/transactions.html", context)
+
```

Comment:
once we migrate, `occurrence_perf_to_email_html` should replace `perf_to_email_html`